template.md

```diff
@@ -56,6 +66,13 @@ Deposit | `/user/deposit_address/{currency_code}`
 Trade | `/user/sell_limit`, `/user/buy_limit`, `/user/cancel_order`
 
 
+# Approved Withdraw Addresses
+
```

Comment:
There should be a warning here about the risks of approving an address. Slate supports this via 'aside' tags:

```
<aside class="warning">
warning here
</aside>
```